scripts/validate_docstrings.py

```diff
@@ -541,7 +542,7 @@ def validate_pep8(self):
         yield from application.guide.stats.statistics_for('')
 
 
-def validate_one(func_name):
+def validate_errs_wrns(func_name, doc):
```

Comment:
`doc` should have an attribute with the `func_name`, I think it's redundant here.